sqllineage/core.py

```diff
@@ -79,9 +79,9 @@ def analyze(self, stmt: Statement) -> LineageResult:
         :param stmt: a SQL statement parsed by `sqlparse`
         """
         if stmt.get_type() == "DROP":
-            self._extract_from_DDL_DROP(stmt)
```

Comment:
I hesitate for a long time that whether we should use upper case or lower case here. Since they're abbreviation, upper case should they be. But it's bad code style to use upper case for method name. Would you share your opinion?

Reply:
Technically, uppercase method names violate PEP 8.  My standard code practice is to try to eliminate any lint warning messages.

Reply:
Sure, let's make it lower case